src/Option_Command.php

```diff
@@ -170,6 +170,24 @@ public function add( $args, $assoc_args ) {
 	 *   - total_bytes
 	 * ---
 	 *
+	 * [--orderby=<fields>]
+	 * : Set orderby which field.
+	 * ---
+	 * default: option_name
```

Comment:
The default orderby behavior is `option_id`, I believe.

Reply:
Yes, you are right. Made some changes according to that.